### PR TITLE
Jupyter R examples

### PR DESCRIPTION
**Pull request by jameno**
I have added the R equivalent data-loading/cgm-stats Jupyter Notebook and .R file for review.

### CODE REVIEW

File: examples/load-data-and-cgm-stats.r [hunk 1]
```diff
@@ -0,0 +1,120 @@
+####################################################################################
+# description: Example for loading Tidepool data and calculating cgm stats in R
+# version: 0.0.1
+# created: 2018-07-17
+# author: Jason Meno
+# license: BSD-2-Clause
+####################################################################################
+
+# Set location of data
+dataPath = "../example-data/"
```
Comment (ed-nykaza) [inline examples/load-data-and-cgm-stats.r]: please use something like file.path() so we can avoid using slashes in the file path. For example, dataPath = file.path("..", "example-data")

File: examples/load-data-and-cgm-stats.r [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+# load csv data
+data = read.csv(paste0(dataPath,
```
Comment (ed-nykaza) [inline examples/load-data-and-cgm-stats.r]: perhaps use file.path here too?

File: examples/load-data-and-cgm-stats.r [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+# convert mmol/L to mg/dL and create a new field
+cgm$mg_dL = as.integer(cgm$mmol_L * 18.01559)
+
+# view the cgm mg/dL data
+head(cgm$mg_dL)
+
+# define a function that captures the Ambulatory Glucose Profile statistics
+# http://www.agpreport.org/agp/agpreports#CGM_AGP
+get_stats <- function(df){
```
Comment (ed-nykaza) [inline examples/load-data-and-cgm-stats.r]: I suggest/prefer to use "=" over "<-" despite the fact "<-" is the legacy R way of doing things.

File: examples/load-data-and-cgm-stats.r (hunk 1 above)
Comment (jameno) [reply examples/load-data-and-cgm-stats.r]: Fixed, now using file.path()

File: examples/load-data-and-cgm-stats.r (hunk 2 above)
Comment (jameno) [reply examples/load-data-and-cgm-stats.r]: Fixed, now using file.path()

File: examples/load-data-and-cgm-stats.r (hunk 3 above)
Comment (jameno) [reply examples/load-data-and-cgm-stats.r]: Fixed, "<-" replaced with "="

**Comment by ed-nykaza** [comment]
@jameno LGTM...please merge and delete when you have a chance

**Pull request merged**